data.sql

```diff
@@ -1,3 +1,4 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES (1,'Argumon',2020-02-03,0,true,10.23), (2,'Gabumon',2018-11-15,2,true,8), (3,'Pikachu',2021-01-07,1,false,15.04), (4,'Devimon',2017-05-12,5,true,11);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) 
+VALUES (1,'Argumon',2020-02-03,0,true,10.23), (2,'Gabumon',2018-11-15,2,true,8), (3,'Pikachu',2021-01-07,1,false,15.04), (4,'Devimon',2017-05-12,5,true,11), (5,'Charmander',2020-02-08,0,false,11), (6,'Plantmon',2021-11-15,2,true,5.7), (7,'Squirtle',1993-04-02,3,false,12.13), (8,'Angemon',2005-06-12,1,true,45), (9,'Boarmon',2005-06-07,7,true,20.4), (10,'Blossom',1998-10-13,3,true,17), (11,'Ditto',2022-05-14,4,true,22);
```

Comment:
- You have done great so far :+1: But there are a few requirements you need to meet before moving to the next milestone. :+1: 

- [ ] Looking at your data, some of the data entered into the DB are not correct, for example, the weight in kg according to the requirement for `Squirtle` should be `-12.13`, it is negative, but yours is not. I suggest you go through the requirements and make sure you insert the correct data into the database. and also rerun your queries because this is going to affect the result. :smile: :wave: